### DISCUSSION TITLE
Can I interact with hosted Budibase DB tables directly?

### DISCUSSION THREAD

**Discussion by equ8r** [Budibase/budibase#5833] [General]
I am hoping to be able to apply sql queries directly... does that require running a local DB, or can I have more direct access using the onboard DB?

**Comment by melohagan** [comment]
Hi @equ8r 

The internal DB is a NoSQL CouchDB. 

If you want to use SQL, you will need an instance of MySQL or PostgreSQL for example. These could be local or hosted on a server.
